### PR TITLE
implement mentor profile

### LINKED ISSUE #256: API: Get profile endpoint for mentor
Create a new endpoint `/api/profile/mentor` that returns profile information for the mentor. The information returned should be:

- name
- organizationId
- organizationName
- timeZone
- skills: array of numbers from `mentor_skills` table

See instructions on the authentication in #255.

### PR DESCRIPTION
**Pull request by jcg-codes**
# Description

This PR is about Profile Endpoint For Mentor

Fixes #256 

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by tpetrina — CHANGES_REQUESTED** [review]
It is a really good start, but you need to return the values mentioned in the issue.

File: src/server/api/controllers/profile.controller.js [hunk 1]
```diff
@@ -18,6 +18,49 @@ const getStudentsProfile = async (userId) => {
...
+    } else {
+      await knex('users').where({ id: userId }).update({
+        name: body.name,
+        timezone: body.timezone,
+      });
+    }
+  } catch (error) {
+    return error.message;
+  }
+};
```
Comment (Elia-Git) [inline src/server/api/controllers/profile.controller.js]: This block of code needs to be removed
1. It wasn't  part of your issue
2.  was aimed to update a student's profile
3. is repeated below from line 65 to 84

**Review by Elia-Git — COMMENTED** [review]
The rest looks good ,just a small fix to get the values mentioned (Toni also mentioned).

File: src/server/api/controllers/profile.controller.js [hunk 2]
```diff
@@ -18,7 +18,28 @@ const getStudentsProfile = async (userId) => {
...
 };
+const getMentorsProfile = async (userId) => {
+  try {
+    const profiles = await knex('users')
+      .select(
+        'users.name',
+        'users.timezone',
+        'organization_id',
+        'organizations.name as organizationName',
+        'skills.name as Skills',
```
Comment (tpetrina) [inline src/server/api/controllers/profile.controller.js]: Did you test this? I find it hard to believe that this is an array. You should probably get skills in a new knex call and you should return numbers:
> skills: array of numbers from mentor_skills table

Also, it should be named `skills` with lowercase `s`.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Good work, but you need to fix how you get skills for the mentor.

**Comment by AnilMula** [comment]
hi Jean , can please push latest changes here

**Comment by tpetrina** [comment]
Please remove changes to package.json and package-lock.json

**Review by tpetrina — APPROVED** [review]
Good job!

**Pull request merged**